azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKWrapper.kt

```diff
@@ -142,6 +149,63 @@ internal class CallingSDKWrapper(
         return endCallCompletableFuture!!
     }
 
+    override fun admitAll(): CompletableFuture<CallCompositeLobbyErrorCode?> {
+        val future = CompletableFuture<CallCompositeLobbyErrorCode?>()
+        val options = AdmitLobbyParticipantOptions()
+        nullableCall?.lobby?.admitAll(options)?.whenComplete { _, error ->
```

Comment:
That should not happen, but what If nullableCall?.lobby? will actually be null, then we will never complete future. 
Should it be completed with error?